bin/fin

```diff
@@ -7709,10 +7710,13 @@ fi
 # out of the box without the need to ask the user to manually configure DNS records using "fin hosts".
 # See https://github.com/docksal/docksal/issues/1276
 if is_wsl && is_docker_native; then
-	# Limit this to Docker Desktop for Windows 2.2.0.0+, which it is absolutely necessary
-	if (( $(ver_to_int $(docker_desktop_version)) >= $(ver_to_int '2.2.0.0') )); then
-		export DOCKSAL_NO_DNS_RESOLVER=1
-		export DOCKSAL_DNS_DOMAIN=docksal.site
+	# Only proceed if DOCKSAL_DNS_DOMAIN was not explicitly set (matches the default)
+	if [[ "${DOCKSAL_DNS_DOMAIN}" == "${DOCKSAL_DNS_DOMAIN_DEFAULT}" ]]; then
```

Comment:
```suggestion
	if [[ "${DOCKSAL_DNS_DOMAIN}" == "" || "${DOCKSAL_DNS_DOMAIN}" == "${DOCKSAL_DNS_DOMAIN_DEFAULT}" ]]; then
```

Reply:
@achekulaev can you help me understand the need for the extra condition?
`DOCKSAL_DNS_DOMAIN` can only be empty if the user sets it so. Even then, `[[ "${DOCKSAL_DNS_DOMAIN}" == "${DOCKSAL_DNS_DOMAIN_DEFAULT}" ]]` check is sufficient IMO.

Reply:
Yeah it's not needed. Missed that case is processed above